index.html

```diff
@@ -55,10 +57,234 @@ <h4>LET'S CONNECT</h4>
+                    <ul class="pop-description-element">
+                        <li class="element" id="companyID">CANOPY</li>
+                        <li class="element">
+                            <img class="bullet" src="./images/bullet.png" alt="bullet" />
+                        </li>
+                        <li class="element" id="workType">Back End Dev</li>
+                        <li class="element">
+                            <img class="bullet" src="./images/bullet.png" alt="bullet" />
+                        </li>
+                        <li class="element" id="date">2015</li>
+                    </ul>
+                </div>
+                <img class="work-pic" src="./images/desktop-images/img1.png" alt="landing page view" />
+                <img class="work-pic pop-img" src="./images/Portoflio_Card/work.png" alt="nature image" />
+                <div class="pop-element">
+                    <p class="description-text descriptionTxt">
+                    </p>
+                    <div class="pop-items">
+                        <ul class="languages">
+
+                            <li class="lang">html</li>
+
+                            <li class="lang1">css</li>
+
+                            <li class="lang2">javaScript</li>
+                        </ul>
+                        <div class="bottom-bar"></div>
+                        <div class="navigation-links">                            
+                        </div>
+                    </div>
+                </div>
+              </div>
+            </div>
+        <section id="works-container">
+            <div class="container">
+                <img class="work-pic" src="./images/Portoflio_Card/work.png" alt="nature image" />
+                <img class="work-pic" src="./images/desktop-images/img1.png" alt="landing page view" />
+                <div class="work-description">
+                    <h4>Tonic</h4>
+                    <ul class="description-element">
+                        <li>CANOPY</li>
+                        <li class="element">
+                            <img class="bullet" src="./images/bullet.png" alt="bullet" />
+                        </li>
+                        <li class="element">Back End Dev</li>
+                        <li class="element">
+                            <img class="bullet" src="./images/bullet.png" alt="bullet" />
+                        </li>
+                        <li class="element">2015</li>
+                    </ul>
+                    <p class="description-tex descriptionTxt">
+                        A daily selection of privately personalized reads; no accounts or
+                        sign-ups required.
+                    </p>
+                    <ul class="languages" id="languages-1">
+                        <li class="lang">html</li>
+                        <li class="lang1">css</li>
+                        <li class="lang2">javaScript</li>
+                    </ul>
```

Comment:
Great job 👍 . Kindly put a space between the description and the language so it can match the Figma design

| Your Design | Figma Design |
| :---:   | :---: | 
|<img width="309" alt="image" src="https://user-images.githubusercontent.com/73854999/219854714-68985c2a-2dcd-4af1-8736-6543a3a1f1dc.png"> | <img width="309" alt="image" src="https://user-images.githubusercontent.com/73854999/219854702-0097a2af-708f-48e8-abd9-50513c6421b0.png">|

Also, kindly use the exact color on your description and other aspects as we have it in the Figma design. ☺️ 🟢